### ISSUE TITLE
logging configuration options

### ISSUE THREAD

**Issue by tizianorosato-gr** [F5Networks/f5-declarative-onboarding#120]
### Is your feature request related to a problem? Please describe.
All logging configuration option should be available.
System  ››  Logs : Configuration : Options

And as matter of priority, the "Audit Logging" setting should be implemented 

### Describe the solution you'd like
All logging options should be configurable from DO.
And the "Audit Logging" settings on top of the others

### Describe alternatives you've considered
none

### Additional context
In a very security resctricted env audit logging should be enabled as soon the device is onboarded, including the GUI that by default is disabled.

**Comment by dstokesf5** [comment]
It looks like many of the logging settings can be configured by modifying data base variables, which Declarative Onboarding supports: 
```json
{
    "schemaVersion": "1.10.0",
    "async": "true",
    "class": "Device",
    "Common": {
    	"class": "Tenant",
    	"dbvars": {
    		"class": "DbVariables",
    		"config.auditing": "disable"
    	}
    }
}
```

However, I cannot figure out how to enable the GUI audit logging through tmsh. Do you happen to know how to configure that setting other than using the management GUI?

**Comment by dstokesf5** [comment]
Finally found the tmsh command for it:
```modify sys global-settings gui-audit enabled```

That cannot currently be configured through Declarative Onboarding.

**Comment by dstokesf5** [comment]
I have created AUTOTOOL-1248 in our internal product backlog for configuring the 3 audit log settings.

**Comment by dstokesf5** [comment]
This issue has been resolved as part of the 1.13.0 release.
